xbmc/cores/VideoPlayer/VideoPlayerRadioID3.h

```diff
@@ -0,0 +1,56 @@
+/*
+ *  Copyright (C) 2005-2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "DVDMessageQueue.h"
+#include "FileItem.h"
```

Comment:
If I see this right, this include is not needed here.

Reply:
DVDMessageQueue.h is required in the header, but FileItem.h is not, and was replaced with `#include "Interface/TimingConstants.h"` for upcoming force-push (and removed from the .cpp), thank you!